source/draw/gpu/opengl/OpenGLPacked.ooc

```diff
@@ -158,6 +158,9 @@ OpenGLPacked: abstract class extends GpuImage {
 			this _backend upload(raster buffer pointer, raster stride)
 		}
 	}
-	onRecycle: func { this _renderTarget invalidate() }
+	onRecycle: func {
+		this referenceCount reset()
```

Comment:
This is needed now as the updated reference counter implementation does not allow to call `free` multiple times on the target. Previously `referenceCount decrease()` called `free()` each time the counter went below 0 (so the opengl images were recycled correctly). And it simply crashed for non-recycled classes (see the updated test). Now it's guarded by atomic flag, so in order to `free()` the same image multiple times we need to reset the counter explicitly when the image is recycled.